src/Sylius/Bundle/OrderBundle/SyliusCartsRemoveEvents.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\OrderBundle;
+
+final class SyliusCartsRemoveEvents
+{
+    const CARTS_PRE_REMOVE = 'sylius.carts.pre_remove';
```

Comment:
PRE_REMOVE

which gives us:

``SyliusExpiredCartsEvents::PRE/POST_REMOVE``

which is much more clear.